site/docs/concepts/web-app.md

```diff
@@ -0,0 +1,272 @@
+You can use either, or both, to work on your Data Flows, depending on your preference.
+
+With the Flow web app, you can perform most common workflows, including:
+
+* Creating end-to-end Data Flows: **capturing** data from source systems and **materializing** it to destinations.
+* Creating, viewing, and editing individual captures and materializations.
+* Viewing data **collections**.
+* Viewing users and permissions.
+* Authenticating with the flowctl CLI.
+
+Some advanced workflows, like granting or revoking permissions and transforming data with **derivations**, aren't available in the web app.
+
+Even if you prefer the command line or plan to perform a task that's only available through flowctl, we recommend you begin your work in the web app;
+it provides a quicker and easier path to create captures and materializations. You can then switch to flowctl to continue working.
+
+## Signing in
+
+You use either a Google or GitHub account to sign into Flow.
+
+![](<./webapp-images/login-screen.png>)
+
+If you've never used Flow before, you'll be prompted to register before being issued a trial account. If you want to use Flow for production workflows or collaborate with team members, you'll need an organizational account.
+[Contact Estuary](mailto:support@estuary.dev) to create a new organizational account or join an existing organization.
+
+## Navigating the web app
+
+When you log into the web app, you land on the **Welcome** page.
+There are four additional pages visible as tabs in the side navigation: **Captures**, **Collections**, **Materializations**, and **Admin**.
+
+The order of the tabs mirrors the order of a basic Data Flow:
+
+import Mermaid from '@theme/Mermaid';
+
+<Mermaid chart={`
+	graph LR;
+		Capture-->Collection;
+        Collection-->Materialization;
+`}/>
+
+While you may choose to [use the tabs in this sequence](../guides/create-dataflow.md), it's not necessary.
+All Flow entities exist individually, outside of the context of complete Data Flow.
+You can use the different pages in the web app to monitor and manage your items in a number of other ways, as described below.
+
+## Captures page
+
+The **Captures** pages shows you a table of existing Flow [captures](./captures.md) to which you have [access](../reference/authentication.md).
+The **New Capture** button is also visible.
+You use the table to monitor your captures.
+
+![](<./webapp-images/capture-page.png>)
+
+**1:** Select all or deselect all.
+
+**2:** Enable, Disable, and Delete buttons. These actions will be applied to the selected table rows. Choose **Disable** to temporarily pause the flow of data, **Enable** to resume, and **Delete** to permanently remove the capture(s).
+
+**3:** Materialize button. When you click this button, you're directed to the **Create materializations** page.
+All the collections of the selected capture(s) will be added to the materialization.
+
+**4:** Filter captures by name.
```

Comment:
I will need to pull this branch down to view the reference images, but I am guessing the reference image is of the full _Captures_ page with numeric labels for its various _segments_. If that assumption is accurate, then the search bar is what is being referenced by the _4_ label.

Granted all the above is true, it is worth highlighting that this search bar allows you to filter captures by catalog name and connector name. Since the catalog names of captures are appended by a connector identifying string (e.g., `/source-firestore`), a connector name query will often match the catalog name in some fashion. However, that is not always the case. For example, the catalog name for a _Google Cloud Storage_ capture is appended by the string `/from-gcs`. While _GCS_ is a commonplace acronym for _Google Cloud Storage_, the explicit connector name (or some variant of it) may be a primary point of reference for a user. A user can easily find all their _Google Cloud Storage_ captures by entering the italicized string (or a subset of it) into the search bar. In the event a user does happen to type a subset of the connector name in the search bar which matches other captures (e.g., typing _Google_ into the search bar can pull captures with an assortment of sources), the table column filters can further facilitate their search.

The same is true for the search bar on the _Materializations_ page.

Reply:
Question here - I mentioned the use of the `*` wildcard in my revision. Is there a list of supported wildcards that I should call out?

Reply:
That is a great question. @travjenkins can you think of any other wildcards worth mentioning?

Reply:
This _is_ a great question... I never even tried using them before honestly. 

We use the following query when typing in the search bar: `${param}.ilike.*${searchQuery}*` which handled by PostgREST and replaces the `*` with `%` on the backend. So I feel like anything that would work in a Postgres `ilike` contained with `%` would work here... but have not manually tested all that.

Reply:
I can come back to this and play around with it another time. I definitely think `*` is the most important one to document.